9 api para periodos POST /periods

serpi90 opened the pull request:
Part of #9

serpi90 committed:
feature: POST /periods

gcotelli commented on source/OKR-Model/PeriodRESTfulControllerSpecification.class.st:
```diff
@@ -42,13 +65,30 @@ PeriodRESTfulControllerSpecification >> getPeriodsRoute [
+	"El nombre es malo a proposito, ya que afecta al content negotiation.
+	Renombrar cuando arreglen el content negotiation en Stargate (https://github.com/ba-st/Stargate/issues/13),
+	Se deja de tal forma a modo de hackazo para que funcione GET /periods con Accept: */* 	(y funcione en los browsers sin definir Accept)
+	 -jmaestri"
```
Eliminar hackazo.. pragmaticamente yo usaría el mismo media type que para no colección

serpi90 replied:
Se cambia para usar el mismo:
- por simplicidad
- hasta que se corriga https://github.com/ba-st/Stargate/issues/13 no vale la pena usar ambos.

serpi90 committed:
fix: use only one mediatype for period

Do not distinguish between singular and plural
- At least until https://github.com/ba-st/Stargate/issues/13 is fixed
- It's simpler
